### ISSUE TITLE
Syntax is not colored

### ISSUE THREAD

**Issue by mbasso** [ahmad2smile/cpx-language#1]
Hi,
I’ve tried to install this plugin on both, windows 10 and Mac OS, but I’m unable to make it work, syntax is not colored. Is there a configuration that I have to do?

**Comment by ahmad2smile** [comment]
No, you just have to switch to CPX. I tested it and it was colored. I check again.

**Comment by mbasso** [comment]
I'll attach here a screenshot of my `vs code@1.23.1` on `macOS High Sierra@10.13.3`, I have selected `CPX` on the bottom and I've disabled all other extensions, to avoid potential conflicts... I've `cpx-language@0.0.1`

![cpx-language](https://user-images.githubusercontent.com/5837614/40131190-91bc9270-5939-11e8-9ae7-35271771ae2a.jpeg)

**Comment by ahmad2smile** [comment]
I reproduced it. Had to disable the cpp extension. I'm busy for now I'll get on it when I can.

**Comment by mbasso** [comment]
great! let me know if I can help
